proposals/roles.md

```diff
@@ -52,14 +58,15 @@ TODO: events?
 
 ### Extension type
 
-An extension type is declared by a non-nested *extension_declaration*.  
-The permitted modifiers on an extension type are `partial`, `unsafe` and 
-the accessibility modifiers `public` and `internal`.  
-Note that `static` is disallowed.  
+An extension type is a role type (new kind of type) declared by an *extension_declaration*.  
+The permitted modifiers on an extension type are `partial`, `unsafe`, `file` and 
+the accessibility modifiers `public`, `internal` and `private`.  
```

Comment:
I thought `file` is an accessibility modifier, like `public`, `internal`, `protected` or `private`

Reply:
`file` affects scope/visibility, but not accessibility. It's a subtle distinction.
The [spec](https://github.com/dotnet/csharplang/blob/main/proposals/csharp-11.0/file-local-types.md) says it's a modifier, not an accessibility modifier.

Reply:
OK, but what's the difference ?
If something is not visible, it's not accessible and if it's not accessible, it's not visible neither.
As `file` is mutually exclusive to accessibility modifier, it seems to be an accessibility modifier as well.

Reply:
> As file is mutually exclusive to accessibility modifier, it seems to be an accessibility modifier as well.

It is only mutually exclusive because we wanted to leave the space open for things like `file private` nested classes, or `file public` implementations of interface members that can be referenced by name in the current file and implement interface members, but not be visible outside the file.

Reply:
Ok, thanks